NuKeeper/Creators/PackageUpdaterCreator.cs

```diff
@@ -21,6 +21,11 @@ public PackageUpdaterCreator(INuKeeperLogger logger, ICreate<IGitHub> githubCrea
 
         public IPackageUpdater Create(SettingsContainer settings)
         {
+            if (settings.UserSettings.ConsolidateUpdatesInSinglePullRequest)
+            {
+                return new ConsolidatingPackageUpdater(_githubCreator.Create(settings), _updateRunner, _logger);
+            }
+
```

Comment:
The one thing that stands out is that we want to move away from using the `ICreate` interfaces where possible. So adding code into them will work against this. Possibly we want to instead lean on the IoC container and give these 2 classes different interface types?